raster-foundry/raster-foundry
# Merge docker-compose.yml with docker-compose.java.yml

## Overview

* Merge `docker-compose.yml` with `docker-compose.java.yml`
* Replace usage of api-server (as an entrypoint to `./sbt`) with sbt service

## Testing Instructions

See CI checks pass.

Destroy your local development environment, and run `setup`:

```bash
$ vagrant destroy -f
$ git clean -dxff
$ ./scripts/setup
```

See that `update` is ran and generates api-server and backsplash assembly JARs.

Ensure tests pass by running `test`:

```bash
$ vagrant ssh
vagrant@vagrant:/opt/raster-foundry$ ./scripts/test
```

Run `server`, and make sure you can interact with the application at http://localhost:9100:

```bash
vagrant@vagrant:/opt/raster-foundry$ ./scripts/server
```
